Task1/Task1.c

```diff
@@ -0,0 +1,75 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include "sortedList.h"
+
+void begin()
```

Comment:
```suggestion
void showHelpMessage()
```
Более информативное название